transformers/integration_tests/median_diss_batch_test.go

```diff
@@ -0,0 +1,87 @@
+package integration_tests
+
+import (
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/lib/pq"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/median_diss/batch"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MedianDissBatch EventTransformer", func() {
+	medianDissConfig := event.TransformerConfig{
+		TransformerName:   constants.MedianDissBatchTable,
+		ContractAddresses: test_data.MedianAddresses(),
+		ContractAbi:       constants.MedianABI(),
+		Topic:             constants.MedianDissBatchSignature(),
+	}
+
+	It("transforms Median diss batch log events", func() {
+		blockNumber := int64(8936530)
+		medianDissConfig.StartingBlockNumber = blockNumber
+		medianDissConfig.EndingBlockNumber = blockNumber
+
+		test_config.CleanTestDB(db)
+
+		header, headerErr := persistHeader(db, blockNumber, blockChain)
+		Expect(headerErr).NotTo(HaveOccurred())
+
+		// TODO: fetch event from blockchain once one exists
```

Comment:
Thought it would be a good idea to have at least a partial integration test in this instance, since the transformer test never actually interacts with the DB, and I wanted to make sure the new Postgres array stuff worked as expected.